scripts/change_url

```diff
@@ -26,31 +26,68 @@ app=$YNH_APP_INSTANCE_NAME
 #=================================================
 ynh_script_progression --message="Loading installation settings..." --weight=1
 
+# Needed for helper "ynh_add_nginx_config"
+final_path=$(ynh_app_setting_get --app=$app --key=final_path)
+
+# Add settings here as needed by your application
 port=$(ynh_app_setting_get --app=$app --key=web_port)
 admin=$(ynh_app_setting_get --app=$app --key=admin)
-final_path=$(ynh_app_setting_get --app=$app --key=final_path)
 
-# Check if the new path stay /SOGo if not exit
+#=================================================
+# BACKUP BEFORE CHANGE URL THEN ACTIVE TRAP
+#=================================================
+ynh_script_progression --message="Backing up the app before changing its URL (may take a while)..."
+
+# Backup the current version of the app
+ynh_backup_before_upgrade
+ynh_clean_setup () {
+	ynh_clean_check_starting
+	# Remove the new domain config file, the remove script won't do it as it doesn't know yet its location.
+	ynh_secure_remove --file="/etc/nginx/conf.d/$new_domain.d/$app.conf"
+
+	# Restore it if the upgrade fails
+	ynh_restore_upgradebackup
+}
+# Exit if an error occurs during the execution of the script
+ynh_abort_if_errors
 
-if [[ $path_url != "/SOGo" ]]
```

Comment:
You, can't use an other than this so why do you remove this ???

Reply:
I think https://github.com/YunoHost-Apps/sogo_ynh/blob/90eaa2a593f6f42a6b286772f6885535058f0fbd/scripts/change_url#L20 do the trick ?

Reply:
Yes well when the user will try to change the path where will have no effect and the user might think that's a bug because it as just no effect.

Maybe you can add a message which explains that changing this is not supported so the user know that it's normal and he can't change this.

Reply:
This should be managed by the core, when the path can't be changed or choose..

The point is: as the new path value is not retrieved, there is no way to add a message or it will be added for anyone.

Reply:
> This should be managed by the core, when the path can't be changed or choose..

well yes, but I dont think that the core implement a way to do this.

> The point is: as the new path value is not retrieved, there is no way to add a message or it will be added for anyone.

Well I don't really understand what did you change because before you change we was able to retrieve the path so why you can't retrieve it know ?